pkg/minikube/cluster/cluster.go

```diff
@@ -624,19 +628,21 @@ func getServiceFromServiceGetter(services serviceGetter, service string) (*kubea
 	return svc, nil
 }
 
-func getServicePortFromServiceGetter(services serviceGetter, service string) (int, error) {
+func getServicePortsFromServiceGetter(services serviceGetter, service string) ([]int, error) {
```

Comment:
I will change so that the all methods use int32, at the time when I made this decision it made sense for the dashboard